src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Components/Actions/EliminationForms/FailedToQualifyForm.razor

```diff
@@ -34,69 +33,57 @@
     [Parameter]
     public FailedToQualify? FailedToQualify { get; set; }
     [Parameter]
-    public EventCallback OnSubmission { get; set; }
+    public EventCallback OnRestore { get; set; }
+    [Parameter]
+    public EventCallback OnEliminate { get; set; }
 
-    private string? FTQReason
+    private IEnumerable<string>? FTQReason { get; set; }
```

Comment:
Does it have to be a property? Also `codes` seems like a better name to me. You're already in FTQ form, no need to provide extra context, unless necessary.

Reply:
Furthermore why not use the enum directly, rather than it's string values?